seleniumByTechnocredits/src/seleniumPractise/pallavi/Alert_Verification.java

```diff
@@ -0,0 +1,41 @@
+package seleniumPractise.pallavi;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+
+public class Alert_Verification 
+{
+
+	public static void main(String[] args) throws InterruptedException 
+	{
```

Comment:
Instead of writing all logic in main method, create separate method for each use case and call those method into main method.